src/db/dbrc.js

```diff
@@ -0,0 +1,50 @@
+import debug from 'debug';
+import db from './db';
+
+const logger = debug('iReporter::initdb:');
+const Models = {
+  recordSchema: `
```

Comment:
is this meant to be `recordSchema` or `userSchema`?

Reply:
should be `userShema`   fixed in #32